winml/lib/Api/LearningModelBinding.cpp

```diff
@@ -150,7 +149,6 @@ void LearningModelBinding::Bind(
     hstring const& name,
     Windows::Foundation::IInspectable const& value,
     Windows::Foundation::Collections::IPropertySet const& properties) try {
-  _winmlt::TelemetryEvent binding_event(_winmlt::EventCategory::kBinding);
 
```

Comment:
This event doesn't send up telemetry. It merely logs a runtime ETW event. This is because there is no telemetry keyword in the TraceLoggingWrite call. An example of telemetry keyword is: MICROSOFT_KEYWORD_MEASURES, MICROSOFT_KEYWORD_TELEMETRY, or MICROSOFT_KEYWORD_CRITICAL_DATA.

I think it would be useful to still have a ETW tracelogging write event that indicates that a binding event in WinML for debugging purposes.